zproject/urls.py

```diff
@@ -560,8 +560,8 @@
     url(r'^api/v1/dev_fetch_api_key$', zerver.views.auth.api_dev_fetch_api_key,
         name='zerver.views.auth.api_dev_fetch_api_key'),
     # This is for fetching the emails of the admins and the users.
-    url(r'^api/v1/dev_get_emails$', zerver.views.auth.api_dev_get_emails,
-        name='zerver.views.auth.api_dev_get_emails'),
+    url(r'^api/v1/dev_get_emails$', zerver.views.auth.api_dev_get_emails_and_realm,
```

Comment:
It might be reasonable to rename.  That said, we might want to keep the old one around for now, so that older versions of the mobile app code can still work with a new server.  

@gnprice what are your thoughts?  I think we care about older servers working with a newer app (for testing) more than we care about older app versions with a newer server.

Reply:
Yeah, if this is for development on the app, we basically don't care about old app versions.

It's also unlikely we care about old server versions either -- note this only applies on a dev server, so the scenario would have to be that we're debugging some issue that only arises on old servers, and to do so we need a dev VM plus the mobile app, and moreover we need different realms on the dev server.

I'd say just rename it. If that unlikely scenario ever arises, we can patch the new name into that dev server then, or the old name into the app -- they'll both be dev versions anyway.

Reply:
How about `dev_list_users`? That feels to me like it better matches the purpose of this route; the exact data involved is more of an implementation detail.